packages/examples/mqtt/iotcc/iotcc_mqtt_edge_system_stats.py

```diff
@@ -0,0 +1,122 @@
+#                                                                             #
+#  The BSD 2-Clause License                                                   #
+#                                                                             #
+#  Redistribution and use in source and binary forms, with or without         #
+#  modification, are permitted provided that the following conditions are met:#
+#                                                                             #
+#  - Redistributions of source code must retain the above copyright notice,   #
+#      this list of conditions and the following disclaimer.                  #
+#                                                                             #
+#  - Redistributions in binary form must reproduce the above copyright        #
+#      notice, this list of conditions and the following disclaimer in the    #
+#      documentation and/or other materials provided with the distribution.   #
+#                                                                             #
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"#
+#  AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE  #
+#  IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE #
+#  ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE  #
+#  LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR        #
+#  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF       #
+#  SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS   #
+#  INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN    #
+#  CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)    #
+#  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF     #
+#  THE POSSIBILITY OF SUCH DAMAGE.                                            #
+# ----------------------------------------------------------------------------#
+
+from liota.core.package_manager import LiotaPackage
+from linux_metrics import cpu_stat,disk_stat,net_stat
+
+dependencies = ["iotcc_mqtt"]
+
+#---------------------------------------------------------------------------
+# User defined methods
+
+
+def read_cpu_procs():
+    return cpu_stat.procs_running()
+
+
+def read_cpu_utilization(sample_duration_sec=1):
+    cpu_pcts = cpu_stat.cpu_percents(sample_duration_sec)
+    return round((100 - cpu_pcts['idle']), 2)
+    
+
+def read_disk_usage_stats():
+    return round(disk_stat.disk_reads_writes('sda')[0], 2)
+
+
+def read_network_bytes_received():
+    return round(net_stat.rx_tx_bytes('ens33')[0], 2)
+
+
+class PackageClass(LiotaPackage):
+
+    def run(self, registry):
+        import copy
+        from liota.entities.metrics.metric import Metric
+
+        # Acquire resources from registry
+        iotcc_edge_system = copy.copy(registry.get("iotcc_edge_system_mqtt"))
```

Comment:
It does not seem to be a good idea to create a whole new package with the same metrics just because the underlying iotcc basic package is using a different protocol channel. Can we make this package (iotcc_edge_system_stats) independent of the underlying iotcc package?